numba_cuda/numba/cuda/cudadrv/nvrtc.py

```diff
@@ -233,12 +232,16 @@ def compile(src, name, cc):
     #   being optimized away.
     major, minor = cc
     arch = f'--gpu-architecture=compute_{major}{minor}'
-    include = f'-I{config.CUDA_INCLUDE_PATH}'
```

Comment:
As mentioned in the comment on cuda_paths.py, there seems to be no substitute for using `CUDA_INCLUDE_PATH`.

Reply:
Due to the issue mentioned [here](https://github.com/NVIDIA/numba-cuda/pull/17#discussion_r1858108925), we probably should not remove the include path here. Because nvrtc can take multiple paths and search them all, the config that user specified through config can still be picked up by rtc.

Reply:
Good point - looks like we can fix this with https://github.com/NVIDIA/numba-cuda/pull/17/files#r1858379047?